scripts/test_cases/msbuild_performance_test_linux.py

```diff
@@ -0,0 +1,171 @@
+"""_summary_
+This script basically runs benchmark test for msbuild
+Returns:
+    None: .....
+"""
+
+import subprocess
+import time
+import os
+import urllib.request
+
+EXTRACT_PATH = "sdk"
+WORKING_DIR = "msbuild-performance-test"
+DOTNET_BASE_VERSION_URL_LINUX = "https://download.visualstudio.microsoft.com/download/pr/dc930bff-ef3d-4f6f-8799-6eb60390f5b4/1efee2a8ea0180c94aff8f15eb3af981/dotnet-sdk-6.0.300-linux-x64.tar.gz"
+DOTNET_DAILY_VERSION_URL_LINUX = "https://aka.ms/dotnet/8.0.1xx/daily/dotnet-sdk-linux-x64.tar.gz"
+TEST_SOLUTION_REPO_URL = "https://github.com/marcin-krystianc/TestSolutions.git"
+TEST_REPO_NAME = "TestSolutions"
+TEST_SOLUTION_CASE = "LargeAppWithPrivatePackagesCentralisedNGBVRemoved"
+TEST_SOLUTION_DIR = "solution"
+
+
+def check_directory(event):
+    """Checking directory for debugging purposes"""
+    print(f'_____CHECKING DIR {event}________')
+    subprocess.call("ls", shell=True)
+
+
+def create_extract_destinations():
+    """ Create the extract destination directories if they do not exist"""
+    check_directory('BEFORE CREATE EXTRACT DESTINATION')
+    if not os.path.exists(EXTRACT_PATH):
+        os.mkdir(EXTRACT_PATH)
+    os.chdir(EXTRACT_PATH)
+    if not os.path.exists("base"):
+        os.mkdir("base")
+    if not os.path.exists("daily"):
+        os.mkdir("daily")
+    check_directory('AFTER CREATE EXTRACT DESTINATION')
+
+
+def download_file(url, filename):
+    """ Download file from url and save it to filename"""
+    check_directory('BEFORE DOWNLOAD FILE')
+    with urllib.request.urlopen(url) as response, open(filename, 'wb') as out_file:
+        data = response.read()
+        out_file.write(data)
+    check_directory('AFTER DOWNLOAD FILE')
+
+
+def download_and_extract_dotnet_sdk(version_url, is_base):
```

Comment:
Instead of boolean argument `is_base`, it could take the name of the destination directory. From the caller point of view it is going to be easier to understand. what the argument does.